Running the plugin as a standalone executable throws Error: Cannot find module 'tslib'

When using the plugin without any other dependency it throws the following error:

```
node:internal/modules/cjs/loader:1073
  throw err;
  ^

Error: Cannot find module 'tslib'
Require stack:
- /Users/workstation/Dev/test_grpc/node_modules/@ngx-grpc/protoc-gen-ng/main.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1070:15)
    at Module._load (node:internal/modules/cjs/loader:923:27)
    at Module.require (node:internal/modules/cjs/loader:1137:19)
    at require (node:internal/modules/helpers:121:18)
    at Object.<anonymous> (/Users/workstation/Dev/test_grpc/node_modules/@ngx-grpc/protoc-gen-ng/main.js:4:15)
    at Module._compile (node:internal/modules/cjs/loader:1255:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1309:10)
    at Module.load (node:internal/modules/cjs/loader:1113:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/workstation/Dev/test_grpc/node_modules/@ngx-grpc/protoc-gen-ng/main.js'
  ]
}

Node.js v20.3.0
```

**Steps to reproduce:**
Add a package.json with the following content:
```json
{
  "devDependencies": {
    "@ngx-grpc/protoc-gen-ng": "^3.1.1"
  },
  "scripts": {
    "proto:generate": "protoc-gen-ng"
  }
}
```
Run from the root directory:
```bash
npm install
npm run proto:generate
```
**Possible solution:**
Add `tslib` as a dependency for package `@ngx-grpc/protoc-gen-ng`

--- a/packages/protoc-gen-ng/package.json
+++ b/packages/protoc-gen-ng/package.json
@@ -14,6 +14,7 @@
   "dependencies": {
     "google-protobuf": "^3.15.8",
     "prettier": "1.19.1",
-    "winston": "^3.2.1"
+    "winston": "^3.2.1",
+    "tslib": "^2.0.0"
   }
 }

